src/components/profilePage/StudentInfo/StudentInfo.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import "./StudentInfoStyles.css";
+
+class StudentInfo extends React.Component {
+    render() {
+        let stuInfo = this.props.studentInfo;
```

Comment:
Make your variable more descriptive don't use short forms. instead of stuInfo you can use studentInfo